File: tutorial/sort/bubble/bbs.py
def BubleSort(list):
    for i in range(len(list)-1):
        for j in range(len(list)-i-1):
            if list[j] > list[j+1]:
                list[j], list[j+1] = list[j+1], list[j]


list = [64, 34, 25, 12, 22, 11, 90]
BubleSort(list)
for i in list:
    print(i)

File: tutorial/sort/bubble/bbs.go
package main

import "fmt"

func BubleSort(arr []int) {
	for i := len(arr) - 1; i > 0; i-- {
		for j := 0; j < i; j++ {
			if arr[j] > arr[j+1] {
				arr[j], arr[j+1] = arr[j+1], arr[j]
			}
		}
	}
}
func main() {
	arr := []int{64, 34, 25, 12, 22, 11, 90}
	BubleSort(arr)
	for i := 0; i < len(arr); i++ {
		fmt.Println(arr[i])
	}
}
